pandas/_libs/tslibs/offsets.pyx

```diff
@@ -437,12 +437,6 @@ class BaseOffset(_BaseOffset):
 # ----------------------------------------------------------------------
 # RelativeDelta Arithmetic
 
-@cython.wraparound(False)
```

Comment:
can you use ``days_per_month_array`` in ``ccalendar.pyx``?

Reply:
> can you use `days_per_month_array` in ccalendar.pyx?

We could import it from the C file, but we can't define it directly in cython.  Since part of the point of ccalendar was to move to pure-cython, I'm not sure we want to re-introduce the C dependency.  Or if we do, we might as well keep the entire implementation in the C file.

Cython 0.28 supposedly allows for "verbatim C" in the form:

```
cdef extern from *:
     """
     foo;
     bar;
     """
```

which could be useful for small snipped.  But when I've tried this I get compile-time errors.